docs/generated/oc_by_example_content.adoc

```diff
@@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
 ====
 
 
+== oc secrets new-sourcecfg
+Create a new source secret
```

Comment:
both server public key and server's host certificate CA go in known_hosts, CA just gets prefixed with `@cert-authority`